src/Pages/Order/Order.js

```diff
@@ -119,7 +126,22 @@ class Order extends Component {
                 this.clickHandlerChangeAutopilotCheckedState
               }
             />
-            <Footer totalPrice={this.state.totalPrice} />
+            {expectPaymentClicked ? (
+              <ExpectPayment
+                clickHandlerChangeExpectPaymentClickState={
+                  this.clickHandlerChangeExpectPaymentClickState
+                }
+              />
+            ) : (
+              ""
+            )}
```

Comment:
```suggestion
            {expectPaymentClicked && (
              <ExpectPayment
                clickHandlerChangeExpectPaymentClickState={
                  this.clickHandlerChangeExpectPaymentClickState
                }
              />
            )}
```
- true인 경우에는 && 연산자로 대체 가능합니다. 기본적인 내용인 만큼 적용하기에 어렵지 않다고 생각합니다.
- 함수 이름이 과도하게 깁니다. 실제 함수 내에 들어있는 코드 내용과 함수의 제목 길이가 비슷하네요. 가독성을 위해서 적당한 길이로 줄여주셔야 합니다.

Reply:
수정완료